cookiecutter-xblock/{{cookiecutter.repo_name}}/Makefile

```diff
@@ -58,9 +54,7 @@ dev.run: dev.clean dev.build ## Clean, build and run test image
 ## Localization targets
 
 extract_translations: symlink_translations ## extract strings to be translated, outputting .po files
-	cd $(PACKAGE_NAME) && i18n_tool extract
-	mv $(EXTRACTED_DJANGO) $(EXTRACTED_TEXT)
-	if [ -f "$(EXTRACTED_DJANGOJS)" ]; then cat $(EXTRACTED_DJANGOJS) >> $(EXTRACTED_TEXT); rm $(EXTRACTED_DJANGOJS); fi
+	cd $(PACKAGE_NAME) && i18n_tool extract --merge-js --combine-partial
```

Comment:
Having spent so much time myself fixing i18n in Open edX, this is much cleaner and gives me joy to read.

Thanks Shadi!